### PR TITLE
argocd: Plugin to perform various actions on ArgoCD

### PR DESCRIPTION
**Pull request by jchikatur**

### CODE REVIEW

**Review by mtkunkel — DISMISSED** [review]

**Comment by jchikatur** [comment]
Docs: https://github.com/walmartlabs/concord-website/pull/122

**Review by mtkunkel — APPROVED** [review]

File: tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/WatchAppResult.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/**
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2021 Walmart Inc.
+ * -----
+ */
```
Comment (benbroadaway) [inline tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/WatchAppResult.java:6]: old license stuck around

File: tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/Application.java [hunk 2]
```diff
@@ -0,0 +1,209 @@
+/**
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2021 Walmart Inc.
+ * -----
+ */
```
Comment (benbroadaway) [inline tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/Application.java:6]: old license stuck around

File: tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/HealthStatus.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/**
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2021 Walmart Inc.
+ * -----
+ */
```
Comment (benbroadaway) [inline tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/HealthStatus.java:6]: old license stuck around

File: tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/SyncStatus.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
+/**
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2021 Walmart Inc.
+ * -----
+ */
```
Comment (benbroadaway) [inline tasks/argocd/src/main/java/com/walmartlabs/concord/plugins/argocd/model/SyncStatus.java:6]: old license stuck around

**Review by benbroadaway — CHANGES_REQUESTED** [review]
apologies for late catch

**Comment by jchikatur** [comment]
> apologies for late catch

removed all of them. Missed them earlier

**Review by benbroadaway — APPROVED** [review]

**Pull request merged**